tools/rosbag/src/record.cpp

```diff
@@ -299,6 +313,5 @@ int main(int argc, char** argv) {
     // Run the recorder
     rosbag::Recorder recorder(opts);
     int result = recorder.run();
-    
```

Comment:
Please revert.

Reply:
Done

Reply:
I actually fixed it in 2d5c41306703c6431e1a399fa600630bec43f641.